public/ga.js

```diff
@@ -0,0 +1,34 @@
+(function() {
+  var _dntStatus = navigator.doNotTrack || navigator.msDoNotTrack;
+  var fxMatch = navigator.userAgent.match(/Firefox\/(\d+)/);
+  var ie10Match = navigator.userAgent.match(/MSIE 10/i);
```

Comment:
Since IE10 and below have been abandoned by Microsoft, do we still care about it? Especially given that no windows 10 computer even comes with it, and even computers that do come with it that are owned by actual people will have been forced onto IE11/Edge by Microsoft through windows update.